Week3/homework/step2-6.js

```diff
@@ -5,12 +5,24 @@ const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function flattenArray2d(arr) {
   // Replace this comment and the next line with your code
-  console.log(arr);
+  const result = [];
+  for (let i = 0; i < arr.length; i++) {
+    for (let j = 0; j < arr[i].length; j++) {
+      result.push(arr[i][j]);
+    }
+  }
+  return result;
 }
 
 function flattenArray3d(arr) {
```

Comment:
These two functions are fit-for-purpose for the arrays provided. Have you thought about how to tackle arrays of any dimension?